packages/itwin/ec3-widget/src/components/EC3/EC3Config.ts

```diff
@@ -0,0 +1,30 @@
+import type { GetAccessTokenFn } from "../api/context/ApiConfigContext";
```

Comment:
`GetAccessTokenFn` is not exposed outside of the package.

Reply:
exported in ```ec3-widget-react.ts```

Reply:
resolved